main.md

```diff
@@ -288,7 +283,30 @@ In the context of a request object as specified in [@I-D.ietf-oauth-jwsreq], `au
 
 Note: Authorization request URIs containing authorization details in a request parameter or a request object can become very long. Implementers SHOULD therefore consider using the `request_uri` parameter as defined in [@I-D.ietf-oauth-jwsreq], potentially in combination with the pushed request object mechanism as defined in [@I-D.lodderstedt-oauth-par] to pass authorization details in a reliable and secure manner.
 
-## Authorization Request Processing 
+Here is an example of a pushed authorization request that sends the authorization request data directly to the AS via a HTTPS-protected connection: 
+
+```
+  POST /as/par HTTP/1.1
+  Host: as.example.com
+  Content-Type: application/x-www-form-urlencoded
+  Authorization: Basic czZCaGRSa3F0Mzo3RmpmcDBaQnIxS3REUmJuZlZkbUl3
+
+  response_type=code&
+  client_id=s6BhdRkqt3
+  &state=af0ifjsldkj
+  &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb 
+  &code_challenge_method=S256
+  &code_challenge=%5B%7B%22type%22:%22https://www.someorg.compayment_ini
```

Comment:
the code_challenge value got cut off or something along with the authorization_details parameter name. 

`&code_challenge=K2-ltc83acc4h0c9w6ESC_rEMTJ3bww-uCHaoeK1t8U
  &authorization_details=%5B%7B%22 ... `